pkg/common/common.go

```diff
@@ -0,0 +1,11 @@
+package common
+
+import (
+	"fmt"
+
+	"github.com/codeready-toolchain/member-operator/pkg/config"
+)
+
+func ToIdentityName(userID string) string {
+	return fmt.Sprintf("%s:%s", config.GetIdP(), userID)
+}
```

Comment:
Just wanted to re-use this func in ctrl and e2e test.  Open for suggestion wrt package_name, file_name etc.

Reply:
It's about IdentityName() and not the actual Identity().  To me it's fine to keep as it for now.  A very minor reusable func and we are keep it public to reuse it anyway.

Reply:
done [9b1b272](https://github.com/codeready-toolchain/member-operator/pull/26/commits/9b1b272ec26e4f6bfef703ca6a6d861e9818f2bb)